src/core/execute.c

```diff
@@ -5147,6 +5148,19 @@ int exec_spawn(Unit *unit,
                 }
         }
 
+        if (context->cgroup_nft_family > 0) {
+                r = nft_set_element_add_uint32(context->cgroup_nft_family,
+                                               context->cgroup_nft_table,
+                                               context->cgroup_nft_set,
+                                               unit->cgroup_id);
+                if (r < 0)
+                        log_info("Adding NFT family %s table %s set %s cgroup %" PRIu64 " failed, ignoring: %m",
```

Comment:
use `log_warning_errno(r, …` here

i.e. always pass the error you actually got. (otherwise %m will print garbage). And use LOG_WARN log level for log messages that indicate some form of failure that we handle gracefully and ignore.